modules/storage/config.go

```diff
@@ -23,15 +22,15 @@ type Config struct {
 	Trace tempodb.Config `yaml:"trace"`
 }
 
-const DefaultBlocklistPoll = 5 * time.Minute
-
 // RegisterFlagsAndApplyDefaults registers the flags.
 func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet) {
 
+	cfg.Trace.BlocklistPollFallback = true
 	cfg.Trace.BlocklistPollConcurrency = tempodb.DefaultBlocklistPollConcurrency
+	cfg.Trace.BlocklistPollTenantIndexBuilders = tempodb.DefaultTenantIndexBuilders
```

Comment:
In the past we've been picky about what we register as flags to prevent "option overload". When a user runs `./tempo --help` we want them to only see the most common options required to get Tempo running and leave most options as settable in yaml only.

I feel like these options should not be CLI params, but am open to discussion.

Reply:
Okay, didn't know about this but makes sense :+1: